src/main/groovy/me/champeau/gradle/JapicmpAbstractTask.groovy

```diff
@@ -93,9 +98,11 @@ abstract class JapicmpAbstractTask extends AbstractTask {
         def options = new Options()
         options.outputOnlyModifications = onlyModified
         options.setAccessModifier(AccessModifier.valueOf(accessModifier.toUpperCase()))
+        options.htmlOutputFile = GOptional.fromNullable(htmlOutputFile?.absolutePath)
```

Comment:
I don't see a simply way to do that without making modifications in japicmp itself. Optional is used there to keep those values:
https://github.com/siom79/japicmp/blob/master/japicmp/src/main/java/japicmp/config/Options.java#L19